packages/release/configured.target

```diff
@@ -1,5 +1,5 @@
 [Unit]
 Description=Bottlerocket user and dynamic configuration complete
-After=settings-applier.service
-Requires=settings-applier.service early-boot-config.service
+After=preconfigured.target
+Wants=preconfigured.target
```

Comment:
```suggestion
After=preconfigured.target
Wants=preconfigured.target
WantedBy=multi-user.target
```

Reply:
I addressed the changed, but `WantedBy` should be in the `[Install]`, section, which you can see underneath this comment.